src/backend/distributed/commands/citus_add_local_table_to_metadata.c

```diff
@@ -1179,9 +1180,29 @@ DropDefaultExpressionsAndMoveOwnedSequenceOwnerships(Oid sourceRelationId,
 	Oid ownedSequenceId = InvalidOid;
 	forboth_ptr_oid(columnName, columnNameList, ownedSequenceId, ownedSequenceIdList)
 	{
-		DropDefaultColumnDefinition(sourceRelationId, columnName);
+		HeapTuple columnTuple = SearchSysCacheAttName(sourceRelationId, columnName);
```

Comment:
can we use `get_attnum (sourceRelationId, columnName)`?

Skip if return value is InvalidAttrNumber

Reply:
Will use `get_attnum()` instead 👍 But I think we don't need to check for `InvalidAttrNumber` since `ExtractDefaultColumnsAndOwnedSequences()` guarantees not returning such an attribute.